de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java

```diff
@@ -246,7 +251,6 @@ protected void disableTypeFilterForHPC() {
             }
         }
         listView.setTypeFilter(filter);
```

Comment:
Just picking a random spot in this file to comment - in the PR I had open, I also had to modify `onAppSelectionChanged`.  Is it no longer needed?  It may have been fixed in a different spot in a different way, so I may have missed it.

Reply:
I believe you are talking about https://github.com/cyverse-de/ui/blob/8548c7b3238e7457da0ac6708a5d09706d785387/de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java#L187 ? That fix is no longer required. But I will double check it.

Reply:
Yup, that's it.  Really, as long as the props don't have Boolean objects anymore, then I think this is fixed. This was a weird spot where things were getting flipped again for some reason.

Reply:
Awesome - I'll close my PR then 👍